public/style.css

```diff
@@ -1,3 +1,14 @@
-<form> {
-	align-items: center;
-}
\ No newline at end of file
+h1,h2 {
```

Comment:
We need to centralise our CSS. There is some CSS on the views HTML.

Reply:
Agreed, another potential piece of learning could be how we create a standard header for elements that appear on most pages (the website title, logout button, etc)

Reply:
Interesting.